### PR TITLE
Facebook notifications, Move to new DB (and use Env vars)

### PR DESCRIPTION
**Pull request by lucashu1**

### CODE REVIEW

File: dist/controller/items.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-"use strict";Object.defineProperty(exports, "__esModule", { value: true });exports.default = void 0;var _config = _interopRequireDefault(require("./../config/config.js"));function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}
+"use strict";Object.defineProperty(exports, "__esModule", { value: true });exports.default = void 0;var _config = _interopRequireDefault(require("./../config/config.js"));
+var _refugee = _interopRequireDefault(require("./refugee.js"));function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}
```
Comment (Raghav-Maheshwari) [inline dist/controller/items.js]: Do you know what this line does? Just curious

File: src/server/controller/refugee.js [hunk 2]
```diff
@@ -14,6 +18,133 @@ function generatePickupCode(itemId) {
...
+    body.entry.forEach(function (entry) {
+
+      // Get the webhook event. entry.messaging is an array, but 
+      // will only ever contain one event, so we get index 0
+      if (entry.messaging) {
+        let message_obj = entry.messaging[0];
+        console.log(message_obj);
+
+        // Log message in SQL
+        let sender = message_obj.sender.id;
```
Comment (Raghav-Maheshwari) [inline src/server/controller/refugee.js]: Whats the reason for logging every message in SQL?

Comment (lucashu1) [reply src/server/controller/refugee.js]: We need a way of retrieving the refugees' User IDs. There's no way to see them from the facebook UI. The IDs that are used in the messages API are "page scoped IDs" (PSIDs), so they're unique to both the refugee and our own FB page.

So I'm thinking that during the onboarding process, we have them send us a message, we get their PSID, and then we can clean up the DB every now and then

Comment (lucashu1) [reply src/server/controller/refugee.js]: The problem with using Heroku logs would be that the logs disappear after a while, so we'd have no way of looking back at a message if it was sent, say, 30 minutes ago

Comment (lucashu1) [reply src/server/controller/refugee.js]: There might be a better way of storing that info though, so open to other ideas too

File: src/server/controller/refugee.js [hunk 3]
```diff
@@ -14,6 +18,133 @@ function generatePickupCode(itemId) {
...
+}
+
+function sendPickupNotification(itemId) {
+  conn.query(
+    "SELECT " +
+    "items.name AS item_name, items.pickup_code, " +
+    "beneficiaries.fb_psid, beneficiaries.first_name, beneficiaries.last_name, " +
+    "stores.name AS store_name " + 
+    "FROM items " + 
+    "INNER JOIN beneficiaries ON items.beneficiary_id = beneficiaries.beneficiary_id " +
```
Comment (Raghav-Maheshwari) [inline src/server/controller/refugee.js]: This kind of SQL query is repeated in a bunch of other places. Moving forward, maybe we want to create internal functions for grabbing items/stores etc. and have our API call those functions? Leaving it up to you if you want to make that refactor in this PR (although maybe we should leave that to a future PR)

File: src/server/controller/refugee.js [hunk 4]
```diff
@@ -14,6 +18,133 @@ function generatePickupCode(itemId) {
...
+          "Your " + rows[0].item_name + " is now available for pickup from " + rows[0].store_name + "!\n" +
+          "Please use pick-up code: " + rows[0].pickup_code;
+        try {
+          messenger.sendTextMessage({
+            id: rows[0].fb_psid,
+            text: message,
+            messaging_type: "MESSAGE_TAG",
+            tag: "SHIPPING_UPDATE"
+          });
+          console.log('Sent successfully');
```
Comment (Raghav-Maheshwari) [inline src/server/controller/refugee.js]: Let's have a more detailed console log here - i.e facebook notification sent to {{refugeeName}}

File: src/server/controller/refugee.js (hunk 2 above)
Comment (Raghav-Maheshwari) [reply src/server/controller/refugee.js]: ok I see - not an issue for now, but yeah we can keep this open to discussion in the future

File: src/server/controller/refugee.js (hunk 4 above)
Comment (lucashu1) [reply src/server/controller/refugee.js]: Good call, I think that was leftover from the FB quick start guide

File: src/server/controller/refugee.js (hunk 3 above)
Comment (lucashu1) [reply src/server/controller/refugee.js]: That would be nice, but I think the 1 issue is how to handle different types of errors (and their corresponding error messages) if we were to extract 1 function from that. That would be a good clean-up item to figure out

File: src/server/controller/refugee.js (hunk 4 above)
Comment (lucashu1) [reply src/server/controller/refugee.js]: Updated!

**Pull request merged**